docs/prover/cli/options.md

```diff
@@ -251,19 +251,6 @@ Whenever you want to use a Solidity compiler executable with a non-default name.
 **Example**  
 `certoraRun Bank.sol --verify Bank:Bank.spec --solc solc8.1`
 
-### `--solc_args`
```

Comment:
I don't understand, was this option removed?  It is still reported in `certoraRun --help`

Reply:
This option was removed. Once merged this option will not be listed in --help